godtools/XMLContent/KGP/kgp-5.xml

```diff
@@ -100,7 +99,7 @@
                 </content:text>
             </paragraph>
             <paragraph>
-                <content:button type="event" value="kgp-6">
+                <content:button type="event" value="kgp-6,send-information-modal">
```

Comment:
Yeah, I would prefer what @frett suggested too. That seems like it will be cleaner in the code.